### PR TITLE
EDU-1241 - Fix incorrect use of "Query" with "List Filter"

### PR DESCRIPTION
**Pull request by stephanos**
## What does this PR do?

The documentation for commands using the `--query` flag link to the wrong page, ie to "Query" instead of "List Filter".

### CODE REVIEW

File: docs/cli/workflow.md [hunk 1]
```diff
@@ -513,7 +514,7 @@ The input file must have a [Workflow ID](/workflows#workflow-id) on each line.
 `temporal workflow reset-batch --input-file=MyInput --input-separator="\t"`
 
 Workflow Executions can also be found by [Query](/workflows#query).
-`temporal workflow reset-batch --query=MyQuery
```
Comment (stephanos) [inline docs/cli/workflow.md:516]: missing ` at the end

File: README.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ This section provides a higher-level view of the change proposal process, partic
 
 **Before proceeding, make sure [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) and [Node.js](https://nodejs.org/en/download/) are installed. Make sure you install the latest version of Node.js (later than 18.0.0).**
 
-On a Mac, use the command `brew install node@16`.
```
Comment (stephanos) [inline README.md:205]: We recommend 18 (see above), so let's suggest that here.

File: README.md [hunk 3]
```diff
@@ -322,6 +322,7 @@ In a separate terminal, run the Worker in the `/assembly` folder.
 
 ```bash
 cd assembly
+yarn install
```
Comment (stephanos) [inline README.md:325]: Without this, you'll get ERR_MODULE_NOT_FOUND

File: docs-src/cli/workflow/cancel.md [hunk 4]
```diff
@@ -19,12 +19,13 @@ A new [Workflow Task](/concepts/what-is-a-workflow-task) will be scheduled, and
 
 `temporal workflow cancel --workflow-id=meaningful-business-id`
 
-In addition to Workflow IDs, Workflows can also be [Signaled](/concepts/what-is-a-signal) by a [Query](/concepts/what-is-a-query).
-`temporal workflow cancel --query=MyQuery`
+In addition to Workflow IDs, Workflows can also be cancelled by a [List Filter](/visibility#list-filter).
```
Comment (stephanos) [inline docs-src/cli/workflow/cancel.md:22]: I couldn't see an obvious link for "cancelled"; is there one/should there be one?

File: docs/dev-guide/python/foundations.md
Comment (stephanos) [inline docs/dev-guide/python/foundations.md:1]: Not sure why this all changed; but `main` sounds right?

**Review by lorensr — APPROVED** [review]
I'll be fixing this in the cli repo as well in the next month, so that it stays fixed whenever we do the next generating-docs-from-CLI

File: docs/dev-guide/python/foundations.md
Comment (flossypurse) [reply docs/dev-guide/python/foundations.md:1]: np, it is from a generated file change - and it should go away once main is merged in here.

File: docs-src/cli/workflow/list.md [hunk 5]
```diff
@@ -11,10 +11,10 @@ tags:
   - workflow list
 ---
 
-The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [Query](/concepts/what-is-a-query).
+The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [List Filter](/visibility#list-filter).
```
Comment (flossypurse) [inline docs-src/cli/workflow/list.md:14]:
```suggestion
The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [List Filter](/concepts/what-is-a-list-filter).
```

Just switching the link to use the id of the file - that way the link automatically updates if the target info moves or changes in anyway.

**Comment by flossypurse** [comment]
> I'll be fixing this in the cli repo as well in the next month, so that it stays fixed whenever we do the next generating-docs-from-CLI

I think the two have diverged so much that I don't think we would ever run that tool again.

**Comment by lorensr** [comment]
Ah gotcha

**Review by flossypurse — APPROVED** [review]

**Comment by stephanos** [comment]
@flossypurse I rebased and re-assembled everything; feel free to merge! (I can't, it says "Merging is blocked")

**Comment by axfelix** [comment]
Thanks, @stephanos!

**Pull request merged**